tests/test_color.py

```diff
@@ -1,8 +1,28 @@
 import pytest
-from manim import Camera, tempconfig, config
+import numpy as np
+
+from manim import Camera, Scene, tempconfig, config
 
 
 def test_import_color():
     import manim.utils.color as C
 
     C.WHITE
+
+
+def test_background_color():
+    import manim.utils.color as C
+
+    S = Scene()
+    S.camera.background_color = C.RED
```

Comment:
**Suggestion/Issue :** Nice Test, but I think it's wrong as it depends on the constant RED, which make this test dependent to another thing not tested here.
Could you, instead of using it, use an hardcoded np.array representing the RED color? 
(same goes for all of the other tests)

Reply:
Good call! I've changed the test to use explicitly specified colors in 7ef9e644 and 619a9473.